what_is_jupyter.md

```diff
@@ -0,0 +1,94 @@
+# What is Jupyter
+
+[jupyter.org/about]: https://jupyter.org/about
+[subprojects]: https://jupyter.org/governance/list_of_subprojects.html
+
+There is a lot in the name *Jupyter*; the project, the software, the notebooks.
+It is not unusual to fill confused with the many terms around "Jupyter".
+But confusion is never nice so let's try to clarify them.
+
+If you are completely new to *Jupyter*, the [`jupyter.org/about`][jupyter.org/about]
+page is good reading for an overview of the **project**.
+Completely open-source, the software developed by the project grew substantially
+as to be composed by many pieces; those *pieces* (Notebook, Lab, Hub)
+are developed by the different Jupyter [**subprojects**][subprojects]
+
+Let's break "Jupyter" in its components so we can clear them apart
+and have a better view of the *software*.
+
+
+## *Notebook* vs *notebook*
+
+Let's start from the most popular concept, what most of the discussions
+about *Jupyter* talk about: the "notebook".
+
+"Notebook" has two meanings: the notebook file format -- the `.ipynb` files --,
+and the Notebook graphical user interface -- the application -- we use to create
+our notebooks.
+
+
+## *Notebook* and *Lab*
+
+There are two graphical user interface (software) to edit notebooks:
```

Comment:
I would remove the dashes. I think we have more than 2 UIs depending on which projects are considered. Should JupyterLite be included?

Reply:
Good question, @willingc . I was willing to include JupyterLite, but was in doubt since it is still under development. Will discuss that tomorrow in the meeting. Thanks!